src/prefect/orion/schemas/actions.py

```diff
@@ -370,11 +373,11 @@ class WorkQueueUpdate(
     schemas.core.WorkQueue.subclass(
         "WorkQueueUpdate",
         include_fields=[
-            "filter",
-            "name",
```

Comment:
is this an acceptable "breaking" change? actions schemas forbid extra fields, so old clients will likely send `name` in the payload and receive 422 responses

Reply:
Ah, good point.